src/rcs_current.yaml

```diff
@@ -0,0 +1,63 @@
+Ceff:
```

Comment:
This file shouldn't be getting generated. It's moved to `params/rcs_current.yaml` 
I think you just need to delete your local copy.